### DISCUSSION TITLE
Guia Python

### DISCUSSION THREAD

**Discussion by AzaelBarbosa** [LaunchX-InnovaccionVirtual/CursoIntroPython#187] [Show and tell]
Buenas a todos, les dejo una Guía que eh estado haciendo de Python en la cual iré añadiendo mas elementos y ejercicios

[Guia Python](https://gray-dinosaur-bcb.notion.site/Python-d4788f5db5df49b7b1c0af3e9f14ffbf)

Espero les sirva y si tienen alguna duda, con gusto puedo ayudarles

Saludos :)

**Comment by ArmandoRiveraT** [comment]
No deja acceder 
sale: You do not have acces to Asgar Notion, Please contact an admin to add you as a member

**Reply by AzaelBarbosa** [reply]
@ArmandoRiveraT Una disculpa amigo ya arregle el link :)
